v3/lints/cabf_smime_br/mailbox_validated_enforce_subject_field_restrictions.go

```diff
@@ -72,12 +72,13 @@ func NewMailboxValidatedEnforceSubjectFieldRestrictions() lint.LintInterface {
 	}
 }
 
-// CheckApplies returns true if the provided certificate contains one-or-more of the following SMIME BR policy identifiers:
+// CheckApplies returns true if the provided certificate is a subscriber certificate and contains one-or-more of the following
+// SMIME BR policy identifiers:
 //   - Mailbox Validated Legacy
 //   - Mailbox Validated Multipurpose
 //   - Mailbox Validated Strict
 func (l *mailboxValidatedEnforceSubjectFieldRestrictions) CheckApplies(c *x509.Certificate) bool {
-	return util.IsMailboxValidatedCertificate(c)
+	return util.IsMailboxValidatedCertificate(c) && util.IsSubscriberCert(c)
```

Comment:
To address https://github.com/zmap/zlint/pull/713#issuecomment-1698616156 raised by @mtgag